src/common/lzma.cpp

```diff
@@ -130,4 +131,67 @@ SeekableReadStream *decompressLZMA1(ReadStream &input, size_t inputSize, size_t
 	return new MemoryReadStream(outputData, outputSize, true);
 }
 
+SeekableReadStream * compressLZMA1(ReadStream &input, size_t inputSize, size_t &outputSize) {
```

Comment:
`SeekableReadStream *compressLZMA1`

And we don't really need that &outputSize parameter, do we? The returned stream already knows its size.